isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -1513,5 +1513,106 @@
+        </p>
+        <p>
+         This bundle also lowers the max iterations to 10 (from default 50). Lowering the max iterations does not affect the bundle solution. However, setting a lower 
+         iteration limit can serve as a flag if you expect your network to bundle quickly. Finally, the sigma0 convergence criteria was not changed from its default 
+         value, it was only explicitly stated in this call.
+        </p>
+      </description>
+      <terminalInterface>
+        <commandLine>
+        jigsaw fromlist=cubes.lis cnet=relative_reg_jig4_edit.net onet=relative_reg_jig5.net update=no file_prefix=jig5 sigma0=1.0e-10 maxits=10 \
+               twist=yes camsolve=accelerations overexisting=yes camera_angles_sigma=.25 camera_angular_velocity_sigma=.1 camera_angular_acceleration_sigma=.01
+        </commandLine>
+        <description>
+        </description>
+      </terminalInterface>
+    </example>
+
+    <example>
+      <brief>Run of jigsaw parameterized for Kaguya Terrain Camera (TC) images and <b>an intermediate ground control</b> network covering the Apollo 15 landing site. </brief>
+      <description>
+        <p>
+         This example jigsaw bundle is run directly after adding ground control points to the previous relative network. With ground control points inserted into the 
+         bundle solution, we can expand the bundle solve parameters to attempt solving for point radius values and the position of the spacecraft.
+        </p>
+        <p>
+         Ground points are weighted more heavily in the bundle than relative control points and adding parameters adds more complexity to the bundle solution. Therefore, it 
+         is common to create and smith a network with only relative points and add ground point in AFTER the relative network (and its bundle solution) is of sufficient quality. 
+         The purpose of this bundle is to ensure the network bundle converges with the added ground points and solve parameters, before committing to updating the camera 
+         pointing on the images.
+        </p>
+        <p>
+         In addition to the previous solve parameters, this bundle turns on the point radius and spacecraft position solve parameters. Radius and spacecraft position 
+         are two parameters which can only be solved for when ground points are present in the bundle solution since they are concerned with absolute positioning.
+         Applying the point radius solve parameter requires the point_radius_sigma to be set. This value is a representation the uncertainty (in meters) of the 
+         cameras pointing corresponding to the correct elevation on the shape model. Often this value can be set, and the appropriateness of the set value can be 
+         checked using the 'POINTS DETAIL' section of the bundleout.txt file output by jigsaw. If the radius total correction of points exceeds the provided sigma, 
+         the uncertainty may need to be increased.
```

Comment:
Be careful correlating these too closely. We've had users confused about the sigmas not being hard constraints. They are a-priori uncertainties, or standard deviations, not hard constraints. So, it's expected that some amount of points have corrections higher than their sigmas, in fact in a perfect world, we'd expect about a third of the points to have corrections higher than 1 sigmas.